lib/constants.js

```diff
@@ -37,3 +37,7 @@ exports.MonsterType = {
 exports.SummonColor = {
 	values: ["beast", "blue", "green", "yellow", "orange", "white", "purple", "pink", "red"],
 };
+
+exports.MonsterName = {
```

Comment:
Looks pretty good. The main thing I'd change is putting this on multiple lines for readability. (I wrapped it at 132 characters since that's fairly common, but I'm not super concerned about the exact number.)

```suggestion
exports.MonsterName = {
	values: ["Ancient Artillery", "Bandit Archer", "Bandit Guard", "Black Imp", "Cave Bear", "City Archer", "City Guard",
		"Cultist", "Deep Terror", "Earth Demon", "Flame Demon", "Frost Demon", "Forest Imp", "Giant Viper", "Harrower Infester",
		"Hound", "Inox Archer", "Inox Guard", "Inox Shaman", "Living Bones", "Living Corpse", "Living Spirit", "Lurker", "Ooze",
		"Night Demon", "Rending Drake", "Savvas Icestorm", "Savvas Lavaflow", "Spitting Drake", "Stone Golem", "Sun Demon",
		"Vermling Scout", "Vermling Shaman", "Wind Demon", "Bandit Commander", "The Betrayer", "Captain of the Guard",
		"The Colorless", "Dark Rider", "Elder Drake", "The Gloom", "Inox Bodyguard", "Jekserah", "Merciless Overseer",
		"Prime Demon", "The Sightless Eye", "Winged Horror", "Aesther Ashblade", "Aesther Scout", "Bear - Drake Abomination",
		"Valrath Tracker", "Valrath Savage", "Wolf - Viper Abomination", "Human Commander", "Valrath Commander",
		"Manifestation of Corruption"],
};
```